sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/sqlstore/util/AbstractHibernateDao.java

```diff
@@ -0,0 +1,29 @@
+package uk.ac.bbsrc.tgac.miso.sqlstore.util;
+
+import org.hibernate.Criteria;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public abstract class AbstractHibernateDao {

```

Comment:
Are the other Hibernate DAOs supposed to extend this? It's currently a dead class.